resources/js/components/Sidebaricon.vue

```diff
@@ -53,7 +51,23 @@
         isOpen: false,
       }
     },
+    computed: {
+      ariaLabel() {
+        if (this.item.attributes.count !== undefined) {
+          return `${this.item.title}, ${this.count} ${this.plural(this.$t('Item'), this.count)}`;
+        } else {
+          return this.item.title;
+        }
+      },
+    },
     methods: {
+      plural(noun, count) {
```

Comment:
I think that it is necessary to translate plurals too . Different languages have different rules to make the plural. For example in Italian letters 'i' or 'e' are used - cane->cani  for dog/dogs, in Spanish 's' or 'es' are used - maní/maníes  for peanut/peanuts, etc. )

Reply:
Fixed in 951571602aa76438dde13930fbaf49e86281921f